kubernetes/kserve/kserve_wrapper/README.md

```diff
@@ -243,25 +243,25 @@ For v1 protocol
 The curl request for inference is as below:
 
 ```bash
-curl -H "Content-Type: application/json" --data @kubernetes/kserve/kf_request_json/v1/bert.json http://127.0.0.1:8080/v1/models/bert:predict
+curl -H "Content-Type: application/json" --data @kubernetes/kserve/kf_request_json/v1/bert.json http://0.0.0.0:8080/v1/models/bert:predict
 ```
 
 The curl request for Explain is as below:
 
 ```bash
-curl -H "Content-Type: application/json" --data @kubernetes/kserve/kf_request_json/v1/bert.json http://127.0.0.1:8080/v1/models/bert:explain
+curl -H "Content-Type: application/json" --data @kubernetes/kserve/kf_request_json/v1/bert.json http://0.0.0.0:8080/v1/models/bert:explain
 ```
 
 For v2 protocol
 
 The curl request for inference is as below:
 
 ```bash
-curl -H "Content-Type: application/json" --data @kubernetes/kserve/kf_request_json/v1/bert.json http://127.0.0.1:8080/v2/models/bert/infer
+curl -H "Content-Type: application/json" --data @kubernetes/kserve/kf_request_json/v1/bert.json http://0.0.0.0:8080/v2/models/bert/infer
```

Comment:
Please update steps to match the KServe docs. User needs to get correct values for SERVICE_HOSTNAME, INGRESS_HOST and INGRESS_PORT to run the actual predictions and explain requests. Steps for BERT model are here:

https://github.com/kserve/kserve/tree/master/docs/samples/v1beta1/torchserve/v2/bert#run-a-prediction